test/ecs/entity.spec.ts

```diff
@@ -48,27 +40,35 @@ describe('Entity container', () => {
   })
 
   it('should fail with creating entity out of range', () => {
-    const realValue = EntityUtils.STATIC_ENTITIES_RANGE[1]
-
-    function changeRange(value: number) {
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore
-      EntityUtils.STATIC_ENTITIES_RANGE[1] = value
-    }
-
-    changeRange(600)
-
     const entityContainer = EntityContainer()
-    const RangeMock = EntityUtils.STATIC_ENTITIES_RANGE
-
-    for (let i = RangeMock[0]; i < RangeMock[1]; i++) {
+    const entitiesAvailable = MAX_U16 - RESERVED_STATIC_ENTITIES
+    for (let i = 0; i < entitiesAvailable; i++) {
       entityContainer.generateEntity()
     }
-
     expect(() => {
       entityContainer.generateEntity()
     }).toThrowError()
 
-    changeRange(realValue)
+    const randomEntityNumber = (32e3 +
+      Math.round(Math.random() * 32e3)) as Entity
```

Comment:
I don't like to have random tests. In my opinion; the tests should be determistic. If you set a fixed seed for the Math.random to repeat the same pattern should be ok.

But this is just a suggestion. Not required.